client/src/Components/Schedule/Schedule.js

```diff
@@ -1,18 +1,67 @@
 import React, { Component } from 'react';
+import fullCalendar from 'fullcalendar';
+import 'fullcalendar/dist/fullcalendar.css';
+import $ from 'jquery';
 import './Schedule.css';
 
 class Schedule extends Component {
+  componentDidMount() {
+    $('#cal').fullCalendar({
+      defaultView: 'agendaWeek',
+      allDaySlot: false,
+      selectable: true,
+      header: {
+        left: '',
+        center: '',
+        right: ''
+      },
+      // Only permit selections within one day
+      selectConstraint:{
+        start: '00:01',
+        end: '23:59',
```

Comment:
This ends up making it so 12:00 AM and 11:30 PM are not options. Do these not count as starting or ending within the same day?